src/store/index.ts

```diff
@@ -17,5 +17,5 @@ export default new Vuex.Store<RootStateI>({
   state: {
     config: {} as EnvConfigI
   },
-  getters
+  getters: getters as any
```

Comment:
Is this needed?

Reply:
It is temporarily due to all the added Getters where i define StateIF. 
In the updated index, the getters are typed correctly and not as `any`.